tests/unit/test-cmd/test.build.js

```diff
@@ -150,7 +149,7 @@ describe('build', () => {
         assert.instanceOf(error, UsageError);
         assert.match(
           error.message,
-          /Error .* file .*messages\.json: .*: no such file or directory/);
+          /Error: ENOENT: no such file or directory, open .*messages.json/);
```

Comment:
we should add an assertion that check how our error message starts (e.g. using a regexp like `/^Error reading messages.json/`) and also check that the full path of the messages.json file is included in the error message (like the `assert.include(error.message, messageFileName)` above